admin/src/pages/SettingsPage/index.tsx

```diff
@@ -205,7 +205,7 @@ const SettingsPage = () => {
       };
     }
     return ct;
-  }) : [];
+  }), ct => ct.info.displayName) : [];
```

Comment:
I will be more than happy, if You could explain how this works ;)

Reply:
This second argument for `sotBy` is the iteratees to sort by. So I'm just sorting by displayName of each content type. This could also be:

```suggestion
  }), ["info.displayName"]) : [];
```